### PR TITLE
Archive synapse samples, update readme to remove "Modern Data Warehouse" terminology, replace with "Medallion"

### PR DESCRIPTION
**Pull request by devlace**
# Type of PR

- Documentation changes

## Purpose

- Updates the main readme to remove references to "Modern Data Warehouse" in preparation for repo rename. 
- Tidied up README
- Updated diagram to drawio
- Archived Synapse samples

## Author pre-publish checklist

- [ ] Added test to prove my fix is effective or new feature works
- [X] No PII in logs
- [X] Made corresponding changes to the documentation

## Validation steps

- ...

## Issues Closed or Referenced

- Closes #issue_number

- References #issue_number

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,33 +29,25 @@ The samples are either focused on a single azure service (**Single Tech Samples*
...
+| [Azure Data Factory](./single_tech_samples/datafactory/README.md) | ▪️ [CI/CD for ADF with Auto publish](./single_tech_samples/datafactory/adf_cicd_auto_publish/README.md)<br> ▪️ [Data pre-processing using Azure Batch](./single_tech_samples/datafactory/adf_data_pre_processing_with_azure_batch/README.md) |
+| [Azure Stream Analytics](./single_tech_samples/streamanalytics/README.md) | ▪️ [CI/CD for Azure Stream Analytics](./single_tech_samples/streamanalytics/streamanalytics_ci_cd/README.md) |
 
 ## End to End samples
 
-### Parking Sensor Solution
+### DataOps for Medallion with Azure Data Factory and Azure Databricks
 
-This demonstrates batch, end-to-end data pipeline following the MDW architecture, along with a corresponding CI/CD process.
+This [sample](e2e_samples/parking_sensors/) demonstrates batch, end-to-end data pipeline utilizing Azure Data Factory and Azure Databricks built according to the Medallion architecture, along with a corresponding CI/CD process, Observability and automated testing.
```
Comment (promisinganuj) [inline README.md:44]: We can explain in 2-3 sentences what Medallion architecture is.
or
At least, provide hyperlink to the doco:
[Medallion architecture](https://learn.microsoft.com/en-us/azure/databricks/lakehouse/medallion)

**Comment by promisinganuj** [comment]
Can we also introduce the concept of "utilities" in this PR? We can copy a script from this location, if required:
https://github.com/Azure-Samples/modern-data-warehouse-dataops/tree/main/single_tech_samples/fabric/fabric_ci_cd#other-useful-utility-scripts

File: README.md (hunk 1 above)
Comment (devlace) [reply README.md]: Added

**Comment by devlace** [comment]
Added the utilities folder but did not move your script. We can do that as a separate PR.

**Review by bsherwin — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -13,14 +13,14 @@ products:
...
 ---
 
-# DataOps for the Modern Data Warehouse
+# DataOps
 
-This repository contains numerous code samples and artifacts on how to apply DevOps principles to data pipelines built according to the [Modern Data Warehouse (MDW)](https://learn.microsoft.com/en-au/azure/architecture/solution-ideas/articles/enterprise-data-warehouse) architectural pattern on Microsoft Azure.
+This repository contains numerous code samples and artifacts on how to apply DevOps principles to common data engineering patterns and architectures utilizing Microsoft Data Platform technologies.
 
-The samples are either focused on a single azure service (**Single Tech Samples**) or showcases an end to end data pipeline solution as a reference implementation (**End to End Samples**). Each sample contains code and artifacts relating one or more of the following
+The samples are either focused on a single azure service (**Single Tech Samples**) or showcases an end to end data pipeline solution as a reference implementation (**End to End Samples**). Each sample contains code and artifacts relating one or more of the following capabilities:
```
Comment (promisinganuj) [inline README.md:23]:
```suggestion
The samples are either focused on a single microsoft service (**Single-Technology Samples**) or showcases an end-to-end data pipeline solution as a reference implementation (**End-to-End Samples**). Each sample contains code and artifacts related to one or more of the following capabilities:
```

File: README.md [hunk 3]
```diff
@@ -29,33 +29,25 @@ The samples are either focused on a single azure service (**Single Tech Samples*
 
 ## Single Technology Samples
```
Comment (promisinganuj) [inline README.md:30]:
```suggestion
## Single-Technology Samples
```

File: README.md [hunk 4]
```diff
@@ -29,33 +29,25 @@ The samples are either focused on a single azure service (**Single Tech Samples*
...
+| Technology | Samples |
+| ---------- | ------- |
+| [Microsoft Fabric](./single_tech_samples/fabric/README.md) | ▪️ [CI/CD for Microsoft Fabric](./single_tech_samples/fabric/fabric_ci_cd/README.md)<br> ▪️ [Feature engineering on Microsoft Fabric](./single_tech_samples/fabric/feature_engineering_on_fabric/README.md) |
+| [Azure SQL database](./single_tech_samples/azuresql/README.md) | ▪️ [CI/CD for Azure SQL database](./single_tech_samples/azuresql/azuresql_ci_cd/README.md) |
+| [Azure Databricks](single_tech_samples/databricks/) | ▪️ [CI/CD for Azure Databricks](single_tech_samples/databricks/databricks_ci_cd/README.md) |
+| [Azure Data Factory](./single_tech_samples/datafactory/README.md) | ▪️ [CI/CD for ADF with Auto publish](./single_tech_samples/datafactory/adf_cicd_auto_publish/README.md)<br> ▪️ [Data pre-processing using Azure Batch](./single_tech_samples/datafactory/adf_data_pre_processing_with_azure_batch/README.md) |
+| [Azure Stream Analytics](./single_tech_samples/streamanalytics/README.md) | ▪️ [CI/CD for Azure Stream Analytics](./single_tech_samples/streamanalytics/streamanalytics_ci_cd/README.md) |
 
 ## End to End samples
```
Comment (promisinganuj) [inline README.md:40]:
```suggestion
## End-to-End Samples
```

File: README.md [hunk 5]
```diff
@@ -29,33 +29,25 @@ The samples are either focused on a single azure service (**Single Tech Samples*
...
 
-### Parking Sensor Solution
+### DataOps for Medallion with Azure Data Factory and Azure Databricks
 
-This demonstrates batch, end-to-end data pipeline following the MDW architecture, along with a corresponding CI/CD process.
+This [sample](e2e_samples/parking_sensors/) demonstrates batch, end-to-end data pipeline utilizing Azure Data Factory and Azure Databricks built according to the [Medallion architecture](https://learn.microsoft.com/en-us/azure/databricks/lakehouse/medallion), along with a corresponding CI/CD process, Observability and automated testing.
```
Comment (promisinganuj) [inline README.md:44]:
```suggestion
This [sample](e2e_samples/parking_sensors/) demonstrates batch, end-to-end data pipeline utilizing Azure Data Factory and Azure Databricks built according to the [medallion architecture](https://learn.microsoft.com/en-us/azure/databricks/lakehouse/medallion), along with a corresponding CI/CD process, observability and automated testing.
```

File: README.md [hunk 6]
```diff
@@ -13,14 +13,14 @@ products:
...
 - azure-synapse-analytics
-description: "Code samples showcasing how to apply DevOps concepts to the modern data warehouse architecture leveraging different Azure data technologies."
+description: "Code samples showcasing how to apply DevOps concepts to common data engineering patterns and architectures leveraging different Microsoft data platform technologies."
 ---
 
-# DataOps for the Modern Data Warehouse
+# DataOps
 
-This repository contains numerous code samples and artifacts on how to apply DevOps principles to data pipelines built according to the [Modern Data Warehouse (MDW)](https://learn.microsoft.com/en-au/azure/architecture/solution-ideas/articles/enterprise-data-warehouse) architectural pattern on Microsoft Azure.
+This repository contains numerous code samples and artifacts on how to apply DevOps principles to common data engineering patterns and architectures utilizing Microsoft Data Platform technologies.
```
[the remark this answers is not in the document]
Comment (devlace) [reply README.md]: Done

**Pull request merged**